bbdo/neb.proto

```diff
@@ -615,9 +615,8 @@ message Acknowledgement {
   uint64 service_id = 2;
   uint64 instance_id = 3;
   enum AcknowledgementType {
-    NONE = 0;
-    NORMAL = 1;
-    STICKY = 2;
+    HOST = 0;
+    SERVICE = 1;
```

Comment:
Could we have an enum on engine side to be sure we did not miss something...
I browsed the code and saw several comments talking about acknowledgement types as NONE, NORMAL or STICKY.
So with an enum we will be sure to make the correction everywhere.

Reply:
replaced by an enum